unimport/constants.py

```diff
@@ -1,7 +1,39 @@
 import sys
+import typing
 
-PY38_PLUS = sys.version_info >= (3, 8)
 DESCRIPTION = (
     "A linter, formatter for finding and removing unused import statements."
 )
 VERSION = "0.3.0"
+
+PY38_PLUS = sys.version_info >= (3, 8)
+PY37_PLUS = sys.version_info >= (3, 7)
```

Comment:
We have a special case for `PY38_PLUS`, we have one for `PY36`, do we need this, and `PY36_PLUS` *just* so that we can write `PY36`? Why not just delete this and `PY36_PLUS` and declare `PY36` as `sys.version_info==3, 6`